packages/node-core/src/indexer/testing.service.ts

```
@@ -52,6 +52,10 @@ export abstract class TestingService<A, SA, B, DS> {
     protected readonly apiService: IApi<A, SA, B>,
     protected readonly indexerManager: IIndexerManager<B, DS>
   ) {
+    if (!nodeConfig.disableHistorical) {
+      throw new Error(`Historical data not supported in testing service`);
```

I don't think we need to throw an error. Just set it to null